tests/Doctrine/Tests/ORM/Functional/Ticket/DDC6613Test.php

```diff
@@ -0,0 +1,107 @@
+<?php
```

Comment:
This entire test should probably be ported to `PersistentCollectionTest` and stripped of any `UnitOfWork` interactions